queue/src/fiskaltrust.Middleware.Localization.QueueDE/Repositories/DailyClosingRepository.cs

```diff
@@ -76,8 +76,8 @@ public async Task<ftQueueItem> GetQueueItemOfMissingIdAsync(ftActionJournal acti
             var receiptNumerator = JsonConvert.DeserializeAnonymousType(actionJournal.DataJson, new { ftReceiptNumerator = 0L }).ftReceiptNumerator;
 
             // QueueItems are stored before the ActionJournals, so we need to look for the QueueItem with the closest timestamp before the respective ActionJournal
-            var to = new DateTime(actionJournal.TimeStamp).AddMinutes(10).Ticks;
-            var queueItemsInRange = _queueItemRepository.GetByTimeStampRangeAsync(actionJournal.TimeStamp, to);
+            var to = new DateTime(actionJournal.TimeStamp).AddDays(1).Ticks;
```

Comment:
This will still get all queue items within the range of 24 hours, and then you just take the first 10, right? Do we have those extreme cases where queueitem and actionjournal upload differ in 24h?

Reply:
yes thats the problem here, Novacom is manipulating timestamps